spec/fixtures/vcr_cassettes/UnpaywallPublicationImporter/_import_all/when_an_existing_publication_does_not_have_a_DOI/does_not_create_any_open_access_locations_for_the_publication.yml

```diff
@@ -0,0 +1,213 @@
+---
```

Comment:
I think some of these vcr cassettes can be deleted, like this one.  They might have been created when the tests were ran after the importers were changed but the tests weren't changed.  Looks like there are about 28 unpaywall tests that send requests, so there should be about the same number of vcr cassettes.